wix/README.md

```diff
@@ -5,11 +5,13 @@ The WIX toolset is required to build the Windows Installer file, the WIX toolset
 It can be downloaded from http://wixtoolset.org.
 
 ## Build Instructions
+Update the dependencies.wxs file by running ./generate-dependencies.sh (in mingw64 bash shell).
+
 Run candle.exe to build wixobj file from the wxs file:
-candle.exe -arch x64 windows-connector.wxs
+candle.exe -arch x64 windows-connector.wxs dependencies.wxs
```

Comment:
Need full path to candle.exe and light.exe here. I've installed them, but they aren't on my path.